examples_tests/22.RaytracedAO/raytraceCommon.glsl

```diff
@@ -252,14 +260,82 @@ vec3 load_normal_and_prefetch_textures(
 	return geomNormal;
 }
 
+vec3 nbl_glsl_unormSphericalToCartesian(in vec2 uv, out float sinTheta)
```

Comment:
`nbl_glsl_sampling_envmap_directionFromUVCoord` and put it in the header.

Reply:
Done